### ISSUE TITLE
Bug: meetings missing from MeetingBar

### ISSUE THREAD

**Issue by sumpg41** [leits/MeetingBar#739]
### macOS version

14.5 (23F79)

### MeetingBar version

4.9.0

### Installation source

AppStore

### Calendars provider

Google Calendar

### Bug description

Please see the screenshot. The "Doubler" meeting is missing from the MeetingBar, and the canceled/deleted "bi-weekly BD meeting" is still showing on MeetingBar

<img width="1080" alt="Screenshot 2024-06-24 at 10 36 17 AM" src="https://github.com/leits/MeetingBar/assets/80375331/eaa725df-477d-468d-84ea-033d7dac31b1">

**Comment by leits** [comment]
Hi @sumpg41,
Thanks for the report!
Could you share your Appearance settings? Maybe some of them are set to hide?

**Comment by sumpg41** [comment]
Hi @leits, here are my appearance settings: <img width="812" alt="Screenshot 2024-06-28 at 7 56 12 AM" src="https://github.com/leits/MeetingBar/assets/80375331/b2c078ee-401a-468f-b275-09cb6aa9bf7a">

This issue is still going on, I have 3 meetings today but only one is showing on MeetingBar
<img width="189" alt="Screenshot 2024-06-28 at 7 58 41 AM" src="https://github.com/leits/MeetingBar/assets/80375331/ed2ac620-4a6f-4566-8d36-23bce52bd036">


<img width="284" alt="Screenshot 2024-06-28 at 7 57 38 AM" src="https://github.com/leits/MeetingBar/assets/80375331/7bfaa5e5-d1fc-4d72-bb8f-b84a9e458078">

**Comment by muratcankaracabey** [comment]
Same for me, a recently added meeting or deleted meeting is not always reflected. The reactiveness of the app for the connected Google calendar seems to be flaky :/

**Comment by sumpg41** [comment]
@leits Quick Actions -> Refresh sources does not fix the problem for me. I am unsure of what to do here

**Comment by aomegax** [comment]
I have the same problem and refresh sources does not solve it.

**Comment by stonebinox** [comment]
Agreed; refresh sources does not help at all. I disconnected the calendar, switched to Mac calendar for a while, switched it back to Google Calendar, manually selected calendars from the listed options in Calendar Preferences of MeetingBar and it still doesn't bring it in. I'm quite sure it worked well about a year ago, not sure if any recent update nuked it. Kinda disappointed since I really did like the app.

**Comment by shimona-usds** [comment]
I feel like it's only showing meetings with zoom/etc and not without. Is that by design? I feel sure it used to show all of them.

**Comment by leits** [comment]
Hi folks,
It looks like the Google Calendar API has changed. I’ve refactored the app for 4.11. Please check it out once the release is live.

**Comment by leocaseiro** [comment]
I had the same issue today with v4.1.6, and restarting the MeetingBar worked, I'll try Quick Actions -> Refresh sources next time.

**Comment by AlexeyRaga** [comment]
I am having similar issues on `4.11.6`.
It feels like it stops refreshing the calendar at some point, and if I don't check / re-change google calendar, it will stop seeing meetings.
I don't know if it is related to some token expiration etc., but the app never warns and never says that something is wrong, just shows no meetings.
